test/testground/go.mod

```diff
@@ -191,14 +191,14 @@ require (
 	go.uber.org/atomic v1.11.0 // indirect
 	go.uber.org/multierr v1.11.0 // indirect
 	go.uber.org/zap v1.24.0 // indirect
-	golang.org/x/crypto v0.18.0 // indirect
+	golang.org/x/crypto v0.19.0 // indirect
 	golang.org/x/exp v0.0.0-20231206192017-f3f8817b8deb // indirect
 	golang.org/x/mod v0.14.0 // indirect
-	golang.org/x/net v0.20.0 // indirect
+	golang.org/x/net v0.21.0 // indirect
```

Comment:
I thought we wanted this to be `golang.org/x/net v0.23.0` to avoid the go vuln warning

Reply:
we doooo, good catch. I fixed in the main go.mod but pretty sure I forgot to run `go work sync` after that

Reply:
https://github.com/celestiaorg/celestia-app/pull/3263/commits/9f19eceb1a5f404854f630780ab128efff7ba41c